config/config.go

```diff
@@ -28,6 +29,15 @@ type Config struct {
 		MinimumPinEntropy float64 `yaml:"minimumPinEntropy,omitempty"`
 		// AllowMonotonousPins will allow Pins like 123456, 122334 or 654321, 662211, ...
 		AllowMonotonicPins bool `yaml:"allowMonotonicPins,omitempty"`
+		// AllowedFromTime is the time in minutes since midnight when the pin is allowed to be used.
```

Comment:
API expects both, I prefer to have explicit midnight from the user than using the golang 0 value as default. But can do that too if you like.

Reply:
whatever you prefer. I just want to avoid confusion if an admin configures **only** `AllowedFromTime` and **not** `AllowedUntilTime` and eventually the configured value is ignored.

Reply:
// AllowedUntilTime is the time in minutes since midnight when the pin is allowed to be used.
// Both AllowedFromTime and AllowedUntilTime must be set to enable time restrictions.